content/applications/services/fsm/default_warehouse.rst

```diff
@@ -0,0 +1,40 @@
+===============================
+Set up a user default warehouse
+===============================
+
+Setting up a **default warehouse** can be useful for field technicians who keep a supply in their
+van or those who always resupply in the same warehouse.
+
+With this feature configured, the products in sales orders created during field interventions are
+always pulled from the same warehouse, keeping the inventory accurate.
+
+.. seealso::
+   :doc:`../../inventory_and_mrp/inventory`
+
+Prerequisites
+=============
+
+To set up a user default warehouse, configure the following settings first:
+
+ #. Open the **Inventory** app, then go to :menuselection:`Configuration --> Settings`. Activate
+    **Storage Locations** and **Multi-Step Routes**.
+ #. Have more than one warehouse in your database.
+
+Set up a default warehouse for a user
+=====================================
+
+Go to :menuselection:`Settings --> Users & companies --> Manage users`. Then, choose the user from
+the list.
```

Comment:
Give a little more context here.

Reply:
also specify that you are in the :menuselection: `general settings --> manage users`.